.github/workflows/mobile-ci.yml

```
@@ -1,14 +1,12 @@
 name: Mobile CI
-
 on:
   push:
     branches: [main, master, develop]
     paths:
       - 'mobile/**'
   pull_request:
     branches: [main, master, develop]
-    paths:
-      - 'mobile/**'
+    paths:      - 'mobile/**'
```

actionlint (1.7.12)

[error] could not parse as YAML: block sequence entries are not allowed in this context

(syntax-check)